core/src/main/java/io/grpc/inprocess/InProcessServerBuilder.java

```diff
@@ -33,11 +33,16 @@
 
 import com.google.common.base.Preconditions;
 
+import com.google.common.util.concurrent.MoreExecutors;
```

Comment:
Doh! I fail. Now I don't know what I was looking at to make me think my problem was the same as on master... Will fix and rebase now that master is green.